src/lex/cpp.rs

```diff
@@ -691,28 +784,109 @@ impl<'a> PreProcessor<'a> {
         }
         Ok(())
     }
+    // before:
+    // #define f(a, b, c) a + b + c
+    //           ^
+    // after:
+    // #define f(a, b, c) a + b + c
+    //                   ^
```

Comment:
Can you give some context to this comment?

Reply:
This shows the tokens that will be consumed by this function.

Reply:
Can you write that with the comment? At a first look, i thought this was referring to an error annotation